src/main/java/com/google/sps/data/Accessory.java

```diff
@@ -0,0 +1,57 @@
+package com.google.sps.data;
+
+public class Accessory {
+
+  public enum Type {
+    HAT,
+    GLASSES,
+    COMPANION
+  };
+
+  private String id;
+  private String imageFilePath;
+  private Type type;
+  private int height;
+  private int width;
+  private int xPos;
+  private int yPos;
+
+  public Accessory(
```

Comment:
there should be a space in between Accessory and {

Reply:
I had the linter complain at me with the space, since it is a constructor with parameters.